### ISSUE TITLE
Note8 Oreo 8.0 xposed 90-beta3 crashes FB Messenger

### ISSUE THREAD

**Issue by suirzes** [rovo89/XposedInstaller#457]
After installing Xposed 90-beta3 on Note8 (N950F, Oreo 8.0 API 26, september patches) with TWRP 3.2.3, no verity opt encrypt 6.0,  Magisk 17.1+MagiskManager 6.0 and XposedInstaller 3.1.5,

Facebook Messenger app started crashing when being opened, when opening chats or just plain and simple crashing out of nowhere.
 
[2018-10-09-12-09-27.txt](https://github.com/rovo89/XposedInstaller/files/2459757/2018-10-09-12-09-27.txt)
[2018-10-09-11-32-54.txt](https://github.com/rovo89/XposedInstaller/files/2459758/2018-10-09-11-32-54.txt)
[2018-10-09-11-37-16.txt](https://github.com/rovo89/XposedInstaller/files/2459759/2018-10-09-11-37-16.txt)

Yelp! :)

**Comment by Ibuprophen** [comment]
Is there an XPosed Module for Facebook your using? 


~Ibuprophen

**Comment by suirzes** [comment]
No, only 2 fb-unrelated modules. It crashes messenger in either case - no modules or 2 modules inatalled

**Comment by Ibuprophen** [comment]
To me, the logs just looks like there's a Module that's interacting somehow with the app that's reacting badly... 


~Ibuprophen
